### PR TITLE
New symbolic feature + some essential fixes

### PR DESCRIPTION
**Pull request by ritzvik**

### CODE REVIEW

**Comment by shreyasbapat** [comment]
Do you want this to be backported?

File: src/einsteinpy/utils/christoffel.py [hunk 1]
```diff
@@ -80,6 +80,36 @@ def christoffels(list2d, syms):
     return christlist
 
 
+def simplify_tensor_helper(v):
```
Comment (shreyasbapat) [inline src/einsteinpy/utils/christoffel.py]: Make this private!

File: src/einsteinpy/utils/christoffel.py [hunk 2]
```diff
@@ -80,6 +80,36 @@ def christoffels(list2d, syms):
...
+    ndlist : list
+        N-Dimensional list of sympy expressions representing some tensor
+    
+    Returns
+    -------
+    list
+        N-Dimensional list
+    
+    """
+    return simplify_tensor_helper(ndlist)
```
Comment (shreyasbapat) [inline src/einsteinpy/utils/christoffel.py]: Here too!

File: src/einsteinpy/metric/schwarzschild.py [hunk 3]
```diff
@@ -4,10 +4,9 @@
 import numpy as np
 
 from einsteinpy import constant
-from einsteinpy.integrators import RK45, RK4naive
```
Comment (shreyasbapat) [inline src/einsteinpy/metric/schwarzschild.py]: Great!

File: src/einsteinpy/metric/schwarzschild.py [hunk 4]
```diff
@@ -110,7 +109,7 @@ def from_cartesian(cls, pos_vec, vel_vec, time, M):
     def christ_sym1_00(self, vec):
         num1 = (-2 * _G * self.M.value) + ((_c ** 2) * vec[1])
         num2 = _G * self.M.value
-        deno1 = _c ** 4
+        deno1 = _c ** 2
```
Comment (shreyasbapat) [inline src/einsteinpy/metric/schwarzschild.py]: Change in formula?

File: src/einsteinpy/utils/christoffel.py (hunk 1 above)
Comment (shreyasbapat) [reply src/einsteinpy/utils/christoffel.py]: Also, the parameter, should be more than `v`

Comment (ritzvik) [reply src/einsteinpy/utils/christoffel.py]: > Also, the parameter, should be more than `v`

actually the parameter can be both sympy expression or list. So better keep it ```variable``` or ```v```

File: src/einsteinpy/metric/schwarzschild.py (hunk 4 above)
Comment (ritzvik) [reply src/einsteinpy/metric/schwarzschild.py]: > Change in formula?

The initial sources from which we got metric was faulty. I got the insight while developing Kerr Metric. That's why all scaling factors were needed to be introduced

**Comment by ritzvik** [comment]
> Do you want this to be backported?

What do you mean by this?

**Comment by shreyasbapat** [comment]
Do you want this change in 0.1.1 or directly 0.2.0?

**Comment by ritzvik** [comment]
> Do you want this change in 0.1.1 or directly 0.2.0?

0.1.1 would be good

**Pull request merged**